src/app/[locale]/incident/components/IncidentFormFooter.tsx

```diff
@@ -6,14 +6,19 @@ import { useTranslations } from 'next-intl'
 import { Button } from '@/components/ui/Button'
 import { useStepperStore } from '@/store/stepper_store'
 import { steps, usePathname as usePath, useRouter } from '@/routing/navigation'
+import { ImSpinner8 } from 'react-icons/im'
```

Comment:
Ik zie nu dat er op verschillende plekken vanuit verschillende "subbibliotheken" van react-icons iconen worden geïmporteerd. We hebben het hier over gehad, is denk ik zaak om even te overleggen en te gaan kiezen voor een vaste bibliotheek ofwel eigen iconen.

Reply:
Dit moeten we denk ik even met z'n alle gelijk trekken. Kan eventueel iets later als we gaan kijken naar de algehele styling van de applicatie?

Reply:
Maar we kunnen natuurlijk al wel kiezen voor een icon library ;)

Reply:
Tabler kan een goede optie zijn hiervoor